beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -163,8 +164,14 @@ func (s *Service) processFetchedDataRegSync(
 	blksWithoutParentCount := 0
 	for _, b := range data.bwb {
 		if len(b.Blobs) > 0 {
-			if err := s.cfg.DB.SaveBlobSidecar(ctx, b.Blobs); err != nil {
-				log.WithError(err).Warn("Failed to save blob sidecar")
+			verified, err := verification.BlobSidecarSliceNoop(b.Blobs)
+			if err != nil {
+				log.WithField("root", b.Block.Root()).WithError(err).Error("blobs failed verification")
```

Comment:
If the blob fails verification, why are we still saving it below?

Reply:
good catch, will add a `continue`.

Reply:
oh wait I forgot - if there is an error, `verified` will be nil, and the for loop won't do anything. But I can add a continue anyway since that could be confusing.

Reply:
https://github.com/prysmaticlabs/prysm/pull/13190/commits/d25b7d27edf28cb4e639c33e64a7f766c5b7904e